### PR TITLE
Enabling company tokens in webhook campaign action

### LINKED ISSUE #7615: Contactfield tokens for company fields do not work in campaign webhook actions
## Bug Description
When creating a webhook action in a campaign, you can put tokens in for the values such as {contactfield=email}. This works great for contact level fields, but if you include a field off of the primary company, it comes back blank. 

This should work like tokens in emails, which properly resolve the primary company. 

| Q   | A
| --- | ---
| Mautic version | 2.15.1
| PHP version | 7.2
| Browser | N/A

### Steps to reproduce
1. Create a campaign with a webhook action
2. Within the webhook action, pass a parameter for company zipcode using the token {contactfield=companyzipcode}
3. view payload on the other side of the webhook

 
### Log errors
No errors present in log. It just fails silently.

### PR DESCRIPTION
**Pull request by mabumusa1**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #7615

#### Description:
This replaces the PR #7625

Company field tokens are premissable in various places within Mautic, but not in Webhook campaign actions. If sending data to a third party system, it's likely you'll need to send company data along with contact data.

This PR includes the company data in the contact data to be parsed by the TokenHelper.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a campaign with a webhook action
3. Within the webhook action, pass a parameter for company zipcode using the token {contactfield=companyzipcode}
4. view payload on the other side of the webhook

### CODE REVIEW

**Review by fedys — CHANGES_REQUESTED** [review]
I tested token `{contactfield=companyzipcode}` and it didn't work. Only the following tokens work:
-  `{contactfield=companyname}`
-  `{contactfield=companycity}`
-  `{contactfield=companycountry}`
-  `{contactfield=is_primary}`

You need to add `companyzipcode` (and probably other company fields) into [this query](https://github.com/mautic/mautic/blob/3991c3df7c22cdfdfd1fbc063904cd7027bab90c/app/bundles/LeadBundle/Entity/CompanyRepository.php#L117).

**Comment by escopecz** [comment]
@fedys That's done in https://github.com/mautic/mautic/pull/9270 So do you think we can merge this if that one is merged as well?

**Comment by fedys** [comment]
@escopecz definitely!

**Review by fedys — APPROVED** [review]
After discussion with @escopecz this PR will work properly once  #9270 is merged.

**Review by RCheesley — APPROVED** [review]
Thanks for this PR @mabumusa1 !

Tested using https://webhook.site:

screenshot-webhook site-2020 11 21-12_52_09 (2)

Confirmed without the PR applied that the company name field is empty (used this rather than zip code based on the comments above).

screenshot-webhook site-2020 11 21-12_52_09 (1)

After applying the PR the company name comes through as expected and matches the company assigned to the contact:

screenshot-webhook site-2020 11 21-12_52_09

**Pull request merged**
